src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -1790,5 +1790,301 @@ class B
 
             Test(code, expected, index: 0, compareTokens: false);
         }
+
+        [WorkItem(528, "github.com/dotnet/roslyn/issues/528")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        public void TestIntroduceLocalInExpressionBodiedMethod()
+        {
+            var code =
+    @"using System;
+class T
+{
+    int m;
+    int M1() => [|1|] + 2 + 3 + m;
+}";
+
+            var expected =
+    @"using System;
+class T
+{
+    int m;
+    int M1()
+    {
+        const int {|Rename:V|} = 1;
+        return V + 2 + 3 + m;
+    }
+}";
+
+            Test(code, expected, index: 2, compareTokens: false);
```

Comment:
Yeah, this has been kind of broken since they were originally written. That said, at least it's just the index of the refactoring returned by this particular provider. That is, it'll always be zero if it only registers a single refactoring.